src/api.ts

```diff
@@ -487,6 +488,7 @@ class RippleAPI extends EventEmitter {
   rippleTimeToISO8601 = rippleTimeToISO8601 // @deprecated Invoke from top-level package instead
   iso8601ToRippleTime = iso8601ToRippleTime // @deprecated Invoke from top-level package instead
   txFlags = txFlags
+  accountSetFlags = constants.AccountSetFlags
```

Comment:
Do we need to make this `static` in order for the flags to be accessed in the way that the documentation says?